docs/src/pages/LinkPage/LinkPage.js

```diff
@@ -35,8 +35,8 @@ const Page = () => (
       </IntroBlurb>,
     ]}
     webSubpage={<Web wrapped />}
+    iosSubpage={<IOS wrapped />}
     nativeSubpage={<Native wrapped />}
```

Comment:
Your description says you've moved this, but it's still here (as well as on the button page)

Reply:
Yes so this is going to be hidden until the PR https://github.com/Skyscanner/backpack-react-native/pull/521 is merged and this component will take its place for text links 😄

Reply:
How will it be "hidden"?
cos it's showing for me 😆

Reply:
Hmm let me check again locally and see as it was disabled tab before 🤔